tests/plugins/test_bump_release.py

```diff
@@ -129,9 +139,19 @@ def getNextRelease(self, build_info):
             def getBuild(self, build_info):
                 assert build_info['name'] == list(component.values())[0]
                 assert build_info['version'] == list(version.values())[0]
-                if build_info['release'] >= next_release['expected']:
-                    return None
-                return True
+
+                # Check 'major' part of the releases
+                if '.' in next_release['actual'] and \
```

Comment:
Yes, the tests were needlessly complex - now instead of `exists` flag the tests would receive the list of build releases which exist. Changing back to `next_release = self.xmlrpc.getNextRelease(build_info)` fails the tests:
```
next_release = {'actual': '20.1', 'builds': ['20'], 'expected': '21'}
....
assert '20.1' == '21'
```

Tests now also verify that the release is being incremented more than once